functional_tests.go

```diff
@@ -3343,26 +3858,30 @@ func testComposeObjectErrorCasesWrapper(c *minio.Client) {
 	buf := bytes.Repeat([]byte("1"), badSrcSize)
 	_, err = c.PutObject(bucketName, "badObject", bytes.NewReader(buf), "")
 	if err != nil {
-		logger().Fatal("Error:", err)
+		failureLog(function, args, startTime, "", "PutObject failed", err).Fatal()
 	}
 	// 2. Set invalid range spec on the object (going beyond
 	// object size)
 	badSrc := minio.NewSourceInfo(bucketName, "badObject", nil)
 	err = badSrc.SetRange(1, badSrcSize)
 	if err != nil {
-		logger().Fatal("Error:", err)
+		failureLog(function, args, startTime, "", "Setting NewSourceInfo failed", err).Fatal()
 	}
 	// 3. ComposeObject call should fail.
 	if err := c.ComposeObject(dst, []minio.SourceInfo{badSrc}); err == nil {
-		logger().Fatal("Error was expected.")
+		failureLog(function, args, startTime, "", "ComposeObject expected to fail", err).Fatal()
```

Comment:
No point of logging nill error message, err:
failureLog(function, args, startTime, "", "ComposeObject expected to fail"~~, err~~).Fatal()